wicket-core/src/main/java/org/apache/wicket/settings/ExceptionSettings.java

```diff
@@ -208,11 +214,43 @@ public ThreadDumpStrategy getThreadDumpStrategy()
 		return threadDumpStrategy;
 	}
 
-	public NotRenderableErrorStrategy getNotRenderableErrorStrategy() {
+	public NotRenderableErrorStrategy getNotRenderableErrorStrategy()
+	{
 		return notRenderableErrorStrategy;
 	}
 
-	public void setNotRenderableErrorStrategy(final NotRenderableErrorStrategy notRenderableErrorStrategy) {
+	public void setNotRenderableErrorStrategy(
+		final NotRenderableErrorStrategy notRenderableErrorStrategy)
+	{
 		this.notRenderableErrorStrategy = notRenderableErrorStrategy;
 	}
+
+	/**
+	 * Configure noisy Logging in Exceptions
+	 */
+	private boolean loudExceptionLogging;
```

Comment:
Actually this should be `true` by default to preserve the current behavior.

Reply:
yes absolutely, changed.